ui/src/components/flows/Curl.vue

```diff
@@ -5,143 +5,141 @@
         <CopyToClipboard :text="curlCommand" />
 
         <el-alert class="mt-3" type="info" showIcon :closable="false">
-            {{ $t('curl.note') }}
+            {{ t('curl.note') }}
         </el-alert>
     </div>
 </template>
 
-<script>
+<script setup lang="ts">
+    import {computed, ref} from "vue";
     import {baseUrl, basePath, apiUrl} from "override/utils/route";
     import CopyToClipboard from "../layout/CopyToClipboard.vue";
+    import {useI18n} from "vue-i18n";
+    import moment from "moment";
 
-    export default {
-        components: {CopyToClipboard},
-        props: {
-            flow: {
-                type: Object,
-                required: true
-            },
-            inputs: {
-                type: Object,
-                default: () => {}
-            },
-            executionLabels: {
-                type: Array,
-                default: () => []
-            },
-            verbose: {
-                type: Boolean,
-                default: true
-            }
-        },
-        data() {
-            return {
-                exampleFileName: "kestra.json"
-            }
-        },
-        computed: {
-            curlCommand() {
-                const mainCommand = this.generateCurlCommand();
-
-                if (this.flow.inputs && this.flow.inputs.find((input) => input.type === "FILE")) {
-                    return `${this.toShell(this.generatePrefix())} && \\\n${this.toShell(mainCommand)}`;
-                } else {
-                    return `${this.toShell(mainCommand)}`;
-                }
-            },
-            exampleFileInputUrl() {
-                return `https://huggingface.co/datasets/kestra/datasets/resolve/main/json/${this.exampleFileName}`;
+    // Props definition
+    interface FlowInput {
+        id: string;
+        type: string;
+    }
+
+    interface ExecutionLabel {
+        key: string | null;
+        value: string | null;
+    }
+
+    interface Flow {
```

Comment:
Can you use the Flow interface defined in the `flow.ts` store? 
You might have to augment it with `inputs`.

I prefer to avoid defining types in multiple places.